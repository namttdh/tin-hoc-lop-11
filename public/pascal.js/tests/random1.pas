program Sample;
var
    i    : Integer;
    rand : Real;

begin
  for i := 1 to 20 do
  begin
    rand := Random;
    WriteLn('rand = ', rand);
  end;
end.
