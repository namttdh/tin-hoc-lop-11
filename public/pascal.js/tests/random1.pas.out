rand =  3.907985E-14
rand =  9.853947E-04
rand =  4.163100E-02
rand =  1.766426E-01
rand =  3.646022E-01
rand =  9.133061E-02
rand =  9.229765E-02
rand =  4.872172E-01
rand =  5.267503E-01
rand =  4.544334E-01
rand =  2.331784E-01
rand =  8.312918E-01
rand =  9.317315E-01
rand =  5.680596E-01
rand =  5.560943E-01
rand =  5.083191E-02
rand =  7.670512E-01
rand =  1.891480E-02
rand =  2.523597E-01
rand =  2.981972E-01
